test/README

```diff
@@ -18,7 +18,7 @@ Testing requires phantomjs inside the cockpit package.
 To run the integration tests run the following. Don't run the integration tests
 as root:
 
-    $ ./VERIFY
+    $ ./test/verify/run-tests
 
 The first time you run the tests, they will create a cockpit1 bridged network used
```

Comment:
Indeed, this won't work. The tests need the network to run properly, so you need to either add that check into run-tests or document the correct steps.
I would suggest changing the documentation a bit, since that's easier. For initialization, `./test/vm-prep` needs to be run as root to setup the network. Then you can run `./test/verify/run-tests --install` (the install builds the packages) to perform all verify-tests.